.github/workflows/python-tests.yml

```diff
@@ -83,6 +83,24 @@ jobs:
         pip install .[testing,tensorflow]
 
     - run: pytest -sv ./tests/test_keras*
+  
+  build_fastai:
+    runs-on: ubuntu-latest
+
```

Comment:
You will need to add 

```
strategy:
      matrix:
        python-version: ["3.6", "3.9"]
```

as it was broken (and fixed) https://github.com/huggingface/huggingface_hub/commit/7c236b177759e523f6227367b33ac3f936b19638